src/Tooltip/Tooltip.tsx

```diff
@@ -20,77 +22,105 @@ class TargetChildren extends React.Component {
   }
 }
 
-TargetChildren.propTypes = {
+(TargetChildren as any).propTypes = {
```

Comment:
Maybe rather do something like:

```ts
class TargetChildren extends React.Component {
  static proptTypes = {}
}
```

Reply:
@sebald I think that it was done this way to reduce the amont of effort needed for the POC.

Reply:
yes, this is just whatever for now, so we can decide on the style we like.  Even as-is, it's not likely to hurt anything and including it as a static member may just feel like cruft.